src/keychain.js

```diff
@@ -376,65 +373,64 @@ class Keychain {
     if (!validateKeyName(name)) {
```

Comment:
Make the password optional, that is, verify initially on both `exportKey` and `importKey` if password is a function and callback does not exist. If so, assign the callback to the password and password to undefined. It is cleaner to omit the password than having to pass a null parameter. Also add `(optional)` in the js docs please

Reply:
Done!